src/scales/quantitativeScale.ts

```diff
@@ -42,6 +51,44 @@ module Plottable {
       return super.domain(values); // need to override type sig to enable method chaining:/
     }
 
+    /**
+     * Gets the minimum domain value when autoDomain()-ing. 
+     */
+    public min(): D;
+    /**
+     * Sets the minimum domain value when autoDomain()-ing.
+     * 
+     * @returns {QuantitativeScale<D>} The calling QuantitativeScale.
+     */
+    public min(value: D): QuantitativeScale<D>;
+    public min(value?: D): any {
+      if (value === undefined) {
```

Comment:
Having a separate method for clearing the value seems odd, since we don't really do that elsewhere. Maybe we should define a `static` constant to signify no min or max?